update installation-guide.md (#309)
zowe/cics-for-zowe-client: packages/vsce/docs/installation-guide.md

## Installation

### From the VS Code Marketplace

> 1. Open the Extensions tab in the VS Code activity bar.

Reviewer: ```suggestion
1. Open the **Extensions** tab in the VS Code Activity Bar.
```

2. Search for `IBM CICS for Zowe Explorer`.

> 3. Click `Install`. (This will also install `Zowe Explorer` if not already present.).

Reviewer: ```suggestion
3. Click `Install`. (This also installs `Zowe Explorer` if not already present.)
```

> You will now have the Zowe icon in the activity bar with a `CICS` section inside, alongside `Data Sets`, `Unit System Services (USS)` and `Jobs`.

Reviewer: ```suggestion
The **Zowe** icon displays in the Activity Bar, where a **CICS** tree view displays alongside **Data Sets**, **Unit System Services (USS)**, and **Jobs**.
```

### From VSIX File

> 1. Open the Extensions tab in the activity bar.

Reviewer: ```suggestion
1. Open the **Extensions** tab in the Activity Bar.
```

> 2. Click the `...` menu, and press **Install from VSIX...**.

Reviewer: ```suggestion
2. Click the **...** menu, and select**Install from VSIX...**.
```

Author: Adding the space after select.
```suggestion
2. Click the **...** menu, and select **Install from VSIX...**.
```

3. Select the downloaded `Zowe.cics-extension-for-zowe-3.x.x.vsix` file.

<p align="center">
<img src="./images/installing_vsix.gif" alt="Installing IBM CICS for Zowe Explorer" width="700px"/> 
</p>

> The successfull install message should be shown in the bottom right

Reviewer: ```suggestion
The successfull install message should display in the bottom right.
```

<p align="center">
<img src="./images/info-message-install-completed.png" alt="IBM CICS for Zowe Explorer install completed" width="550px"/> 
</p>

> The Zowe Explorer pane will still show tree views for `Data Sets`, `Unit System Services (USS)` and `Jobs`, but in addition a new view `CICS` will be included.

Reviewer: ```suggestion
The Zowe Explorer pane displays tree views for **Data Sets**, **Unit System Services (USS)**, and **Jobs**, but in addition the new **CICS** view is included.
```
